skorch/callbacks/scoring.py

```diff
@@ -198,39 +228,91 @@ class EpochScoring(ScoringBase):
       This is called on y before it is passed to scoring.
 
     """
+    def _initialize_cache(self):
```

Comment:
IMO `initialize -> reset` but `!(reset -> initialize)` and since the method is used for both initialization and reset I think the former is the better name.